tests/projects/test_remote_pipeline.py

```diff
@@ -395,3 +395,20 @@ def test_kfp_pipeline_overwrites_enriched_attributes(
                         assert step["affinity"] == k8s_api.sanitize_for_serialization(
                             tests.projects.assets.remote_pipeline_with_overridden_resources.overridden_affinity
                         )
+
+
+def test_workflow_name():
+    project_name = "test"
+    workflow_name = "test-test"
+    # When creating a schedule workflow we adding the project name to the workflow name.
+    before_renaming = f"{project_name}-{workflow_name}"
+    assert workflow_name == mlrun.utils.normalize_workflow_name(
+        before_renaming, project_name
+    )
+    project_name = "test"
+    workflow_name = "new-1-main"
+    # When creating a schedule workflow we adding the project name to the workflow name.
+    before_renaming = f"{project_name}-{workflow_name}"
+    assert workflow_name == mlrun.utils.normalize_workflow_name(
+        before_renaming, project_name
+    )
```

Comment:
You can use pytest parametrize to test multiple cases instead of duplicating the code :)